dts/st/h7/stm32h723zgtx-pinctrl.dtsi

```diff
@@ -2045,6 +2091,26 @@
 				slew-rate = "very-high-speed";
 			};
 
+			/omit-if-no-ref/ octospim_p1_io2_pc2_c: octospim_p1_io2_pc2_c {
```

Comment:
Sorry, but I'm not sure to follow. If these nodes are not duplicate. Why do they need to be disambiguated ?

Reply:
I was wondering the same but then realized that the the pin name in the device datasheet is actually called PC2_C so I guess this makes sense.

Reply:
> I was wondering the same but then realized that the the pin name in the device datasheet is actually called PC2_C so I guess this makes sense.

yep, that's the point. previous file were actually wrong to my understading.

Reply:
Ok, then this will require an update in release note.